packages/reducers/src/makeReducer.js

```diff
@@ -20,9 +19,9 @@ const createTypeEqualsPredicate = type => compose(whereEq({ type }), nthArg(1));
 const mergeReducers = ([typePredicate, reducer, errorReducer]) => {
 	const newReducer = (state, action) => {
 		if (prop('error', action)) {
-			invariant(errorReducer, `You haven't supplied an error reducer for action ${action.type}`);
-
-			return errorReducer(state, action);
+			if (errorReducer) {
```

Comment:
prefer `if (prop('error', action) && errorReducer)` to nesting ifs

also, please add an entry to changelog (Changed)

```
(_reducers_): `makeReducer` now uses the default reducer for error actions if the error reducer is missing.
```